trustregistry/registry/actor.py

```diff
@@ -0,0 +1,46 @@
+import logging
+from fastapi import APIRouter
+from pydantic import BaseModel, validator
+from typing import List
+from slugify import Slugify
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/registry/actor", tags=["actor"])
+
+custom_slugify = Slugify(to_lower=True, separator="_")
+
+
+class ActorID(BaseModel):
+    id: str
+
+
+class ActorUpdateFields(BaseModel):
+    name: str
+    slug: str = None
+    roles: List[str] = ["issuer", "verifier"]
+    didcomm_invitation: str = None
+    did: str = None
+
+    @validator("slug", always=True)
+    def slugify_id_for_name(cls, value, values):
+        return custom_slugify(values["name"])
+
+
+class RegisterActor(ActorUpdateFields, ActorID):
+    pass
+
+
+@router.post("")
```

Comment:
This is a nitpick, but ("") reads weird to me. Maybe ("/")?
```suggestion
@router.post("/")
```

Reply:
👍